packages/fx-core/src/plugins/resource/frontend/dotnet/plugin.ts

```diff
@@ -111,11 +116,16 @@ export class DotnetPluginImpl implements PluginImpl {
     if (!ctx.projectSettings) {
       throw new NoProjectSettingError();
     }
-    const projectName = ctx.projectSettings!.appName;
-    await scaffoldFromZipPackage(ctx.root, new TemplateInfo({ ProjectName: projectName }));
+
+    const projectName = ctx.answers?.[CoreQuestionNames.AppName];
```

Comment:
If we add tab by `addFeature` command, `ctx.answers` may not contain app name. I think we should get app name from project settings.

Reply:
currently vs does not have "addFeature" command. need to sync with the team to understand the current process of that feature.